test/tools/libtesteth/ImportTest.cpp

```diff
@@ -642,7 +642,7 @@ void ImportTest::checkGeneralTestSectionSearch(json_spirit::mObject const& _expe
 				int errcode = compareStates(expectState, postState, stateMap, WhenError::Throw);
 				if (errcode > 0)
 				{
-					cerr << trInfo << std::endl;
+					cerr << trInfo << "\n" << std::flush;
```

Comment:
Why flush? cerr is not buffered so flush is never needed (am I wrong?). Flush is only needed in some rare cases in cout.